### ISSUE TITLE
Review JSON-QB-API for missing functionality

### ISSUE THREAD

**Issue by RickMoynihan** [Swirrl/graphql-qb#18]
We should review the OGI JSONQB api for any functionality we are missing.

See here for the [currently supported API](https://github.com/OpenGovIntelligence/json-qb-api-implementation)

**Comment by zeginis** [comment]
The functionality of the existing OGI JSON-QB API:
- get all cubes (supported by graphql-qb)
- support aggregations (**ToDo**)
- get cube meta-data (partially suported: title, description, **ToDo**: comment, issued, modified, license, publisher, subject)
- get cube dimensions (supported by graph-ql, **ToDo**: return both a URI and a label)
- get cube attributes i.e. qb.AttributeProperty (**ToDo**)
- get cube measures (**ToDo** see #19)  
- get dimension values (supported by graphql-qb)
- get attribute values (**ToDo**)
- get dimension levels i.e. the hierarchical levels used at a dimension e.g. council area, electoral ward etc (**ToDo**)
- get slice (supported by graphql-qb)
- get table (Not supported. Are we going to use this or the slice is ok?)

**Comment by RickMoynihan** [comment]
> get table (Not supported. Are we going to use this or the slice is ok?)

Yes I think we still want to produce lists of observations that correspond to [these tables](http://statistics.gov.scot/slice?dataset=http%3A%2F%2Fstatistics.gov.scot%2Fdata%2Fearnings&http%3A%2F%2Fpurl.org%2Flinked-data%2Fcube%23measureType=http%3A%2F%2Fstatistics.gov.scot%2Fdef%2Fmeasure-properties%2Fmedian&http%3A%2F%2Fpurl.org%2Flinked-data%2Fsdmx%2F2009%2Fdimension%23refPeriod=http%3A%2F%2Freference.data.gov.uk%2Fid%2Fyear%2F2002&http%3A%2F%2Fstatistics.gov.scot%2Fdef%2Fdimension%2FpopulationGroup=http%3A%2F%2Fstatistics.gov.scot%2Fdef%2Fconcept%2Fpopulation-group%2Fresidence-based) issue #17 is about this.
